mkdocs.yml

```
@@ -55,11 +55,11 @@ nav:
       - Custom Environment Resources: docs/developing/custom-environment-resources.en.md
       - Domain: docs/developing/domain.en.md
       - Internal Load Balancers: docs/developing/internal-albs.en.md
+      - Internal Traffic: docs/developing/internal-traffic.en.md
```

nit: I wonder if "Internal Traffic" is clear enough for folks? I think the "Service Discovery" name is the typical name for this concept: https://microservices.io/patterns/server-side-discovery.html alternatively, we can do:
```suggestion
      - Service-to-Service Communication: docs/developing/internal-traffic.en.md
```

---

if we change it, we should also maybe change the file name?